Header header
uint32 numConstraints
string[] jobID
time[] startTimes
time[] endTimes
uint32[] priority
